pandas/io/formats/style.py

```diff
@@ -2298,6 +2298,7 @@ def set_sticky(
                             "selector": f"thead tr:nth-child({obj.nlevels+1}) th",
                             "props": props
                             + (
+                                # pylint: disable=undefined-loop-variable
                                 f"top:{(i+1) * pixel_size}px; height:{pixel_size}px; "
```

Comment:
I don't understand what's going on in this file. The `i` causing the warning is defined on line 2284.

Reply:
it's just that it's outside of the loop